haproxy/tests/test_haproxy.py

```diff
@@ -264,6 +277,21 @@ def test_version_metadata_tcp_socket(check, version_metadata, datadog_agent):
     datadog_agent.assert_metadata_count(metadata_count)
 
 
+@pytest.mark.usefixtures('dd_environment')
+@pytest.mark.integration
+@pytest.mark.skipif(
+    os.environ.get('HAPROXY_VERSION', '1.5.11').split('.')[:2] < ['1', '7'] or not platform_supports_sockets,
```

Comment:
Let's turn the version condition into a function like `haproxy_version_less_than_1_7`